voltron/plugins/view/memory.py

```diff
@@ -85,13 +88,21 @@ def render(self, results):
                     chunk = m_res.memory[c:c + self.args.bytes]
                     addr_str = self.colour(self.format_address(m_res.address + c, size=target['addr_size'], pad=False),
                                            self.config.format.addr_colour)
+
+                    if self.args.track and self.last_memory and self.last_address == m_res.address:
+                        bytes = []
+                        for i, x in enumerate(six.iterbytes(chunk)):
+                            if x != self.last_memory[c + i]:
+                                bytes.append(self.colour("%02X" % x, 'red'))
```

Comment:
Could use a configuration colour here, but the one used in the registers view for tracking changes didn't seem to work here for some reason.